Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM ubuntu:14.04
  
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
+RUN apt-get upgrade
```

Comment:
I don't think upgrading image from inside is a good idea. If user interested in fresh version of image he should do ``docker pull`` and fetch one. See https://docs.docker.com/articles/dockerfile_best-practices/ for ``upgrade``.